pyproject.toml

```diff
@@ -11,3 +11,25 @@ line_length = 100
 force_sort_within_sections = true
 # Combines "as" imports on the same line
 combine_as_imports = true
+
+[tool.mypy]
+ignore_missing_imports = true
+show_error_codes = true
+disable_error_code = ["attr-defined", "var-annotated"]
+follow_imports = "skip" # Don't follow imports into other files. This should be removed once all type errors have been resolved.
+exclude = [
+    "^dandiapi/api/admin.py",
+    "^dandiapi/api/mail.py",
+    "^dandiapi/api/tests/",
+    "^dandiapi/api/management/",
+    "^dandiapi/api/migrations/",
+    "^dandiapi/api/models/",
+    "^dandiapi/api/tasks/",
+    "^dandiapi/api/views/",
+]
```

Comment:
For now, I've excluded all files and directories that have type errors. I'll make follow up PRs where I fix the type errors in each of these and remove the corresponding entry from `exclude`.